src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/action/PerformanceAnalyzerActionListenerTests.java

```diff
@@ -15,44 +15,52 @@
 
 package com.amazon.opendistro.elasticsearch.performanceanalyzer.action;
 
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.config.PluginSettings;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.metrics.PerformanceAnalyzerMetrics;
+import org.junit.Before;
 import org.junit.Test;
+
 import static org.junit.Assert.assertEquals;
 
 public class PerformanceAnalyzerActionListenerTests {
 
+    @Before
+    public void setup() {
+        PluginSettings.instance().setMetricsLocation("/tmp/");
```

Comment:
Can we write to some temporary location within our repo instead of /tmp?

Reply:
writing to build/tmp/junit_metrics instead of /tmp.